content/tbb/how-to-verify-signature/contents.lr

```diff
@@ -51,13 +51,13 @@ Import the Tor Browser Developers signing key (0xEF6E286DDA85EA2A4BA7DE684E2C6E8
 
 This should show you something like:
 
-    gpg: key 4E2C6E8793298290: public key "Tor Browser Developers (signing key) <torbrowser@torproject.org>" imported
+    gpg: key 4E2C6E8793298290: "Tor Browser Developers (signing key) <torbrowser@torproject.org>" 2 new signatures
     gpg: Total number processed: 1
-    gpg:               imported: 1
-    pub   rsa4096 2014-12-15 [C] [expires: 2020-08-24]
+    gpg:         new signatures: 2
```

Comment:
Same here, you can leave as the original - **imported: 1** 

I added the output of my console, and normally users will see other output.